src/controls/qml/LabeledSwitch.qml

```diff
@@ -20,6 +20,50 @@
 import QtQuick 2.9
 import org.asteroid.controls 1.0
 
+/*!
+    \qmltype LabeledSwitch
+    \inqmlmodule org.controls.asteroid 1.0
+
+    \brief This combines Label and Switch in a convenient package.
+
+    This is a LabeledSwitch.
```

Comment:
Do we need this sentence ? I suppose it'd already be clear from the context ?

Reply:
Good point -- fixed.